userspace/libsinsp/container.cpp

```diff
@@ -240,6 +237,30 @@ string sinsp_container_manager::container_to_json(const sinsp_container_info& co
 	}
 
 	container["metadata_deadline"] = (Json::Value::UInt64) container_info.m_metadata_deadline;
+
+	string json = Json::FastWriter().write(obj);
+
+	/*
+	 * Best effort approach: we got a limit of 64kiB on payload size, see container_to_sinsp_event() lens.
+	 * Try our best to respect it by removing possibly unlimited metadata
+	 */
+	string removableMembers[] = {"port_mappings", "env", "labels", "Mounts" };
+	int idx = 0;
+	Json::Value trimmed_vars = Json::arrayValue;
+	// PPME_CONTAINER_JSON_E now uses EF_LARGE_PAYLOAD flag,
+	// thus its payload can handle up to 4GB of data.
```

Comment:
Practically speaking is there any need to prune the container json object now? I would think 4gb will always be enough.

Reply:
Yup, for me at least there is no much need for this PR anymore; note however that a malicious user could still build json larger than 4G and kill Falco instance; as we are injecting data from outside, i'd go for the safe side and add these checks in any case. 99.999% of the time they won't really matter, but the single time we need them, they do their job.
What do you think?

Reply:
I don't think we need to worry about a malicious container runtime providing intentionally bad responses. And even that happened, this PR wouldn't help, because 4gb per container info is still going to overwhelm the node where falco runs.

Reply:
That's fine for me. Closing the PR then!